src/ovl_61_OpeningScene/29B410.c

```diff
@@ -129,13 +136,25 @@ INCLUDE_ASM(s32, "ovl_61_OpeningScene/29B410", func_800FBB94_2A09C4);
 
 INCLUDE_ASM(s32, "ovl_61_OpeningScene/29B410", func_800FBC9C_2A0ACC);
 
-INCLUDE_ASM(s32, "ovl_61_OpeningScene/29B410", func_800FBCC0_2A0AF0);
+Object* func_800FBCC0_2A0AF0(s32 arg0, void* arg1) {
+    Object* temp_v0;
+
+    temp_v0 = CreateObject(arg0 & 0xFF, arg1);
```

Comment:
This should be able to be written as just `temp_v0 = CreateObject(arg0, arg1);`
This is because the function signature of `CreateObject` is `Object* CreateObject(u8, void*);` so it will do the `& 0xFF` automatically